packages/server/src/auth/auth.resolver.ts

```diff
@@ -13,28 +13,33 @@ import {
   UserSignupDto
 } from './dto/auth.dto';
 import { UsePipes } from '@nestjs/common';
+import { Logger } from '@nestjs/common';
 
 @Resolver()
 export class AuthResolver {
+  private readonly logger = new Logger(AuthResolver.name);
   constructor(private readonly authService: AuthService) {}
 
   /** Login via username */
   @Mutation(() => AccessToken)
   async loginUsername(@Args('user') user: UsernameLoginDto): Promise<AccessToken> {
+    this.logger.log('Username Validated');
```

Comment:
These log messages doesn't make sense. We first entered the function, so it was not validated or successful. I would replace it with something like this.
```suggestion
    this.logger.log('loginUsername Called');
```